### PR TITLE
Adjusting h2 form input for sunetID lookup

### PR DESCRIPTION
**Pull request by lwrubel**
## Why was this change made? 🤔
New SunetID lookup field needs a moment to display and then correct Add button needs to be clicked. Now runs on stage and QA.

### CODE REVIEW

File: spec/features/h2_object_creation_spec.rb [hunk 1]
```diff
@@ -31,7 +31,12 @@
 
     # Adds user to depositor field
     click_button '+ Add another depositor'
+    # wait a second for the SunetID search field to display
+    sleep 1
```
Comment (jcoyne) [inline spec/features/h2_object_creation_spec.rb:35]: I don't understand why this is necessary. Nothing around this changed.

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/features/h2_object_creation_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/features/h2_object_creation_spec.rb:35]: I pulled this branch and commented this line out and the test still works, so can it be removed?

Comment (lwrubel) [reply spec/features/h2_object_creation_spec.rb]: Thanks for testing it and confirming it worked without the sleep line. Have removed. Also at this step, do you know why I might sometimes getting the following and other times it successfully does a lookup? 
Screen Shot 2022-08-22 at 11 12 13 AM

Comment (jcoyne) [reply spec/features/h2_object_creation_spec.rb]: I saw that too once. This is also a problem that we've seen in production before too. I'm not sure they are connected.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**